src/test/java/com/redhat/devtools/lsp4ij/fixtures/LSPClientSideOnTypeFormattingFixtureTestCase.java

```diff
@@ -53,6 +53,7 @@ public abstract class LSPClientSideOnTypeFormattingFixtureTestCase extends LSPCo
 
     public LSPClientSideOnTypeFormattingFixtureTestCase(String... fileNamePatterns) {
         super(fileNamePatterns);
+        setClientConfigurable(true);
```

Comment:
With the new changes to the mock language server definition, tests that need to confirm client config behavior must explicitly opt-in by calling `setClientConfigurable(true);` in the constructor. You'll see several of the base fixtures where I've done that. Any test that _doesn't_ do that doesn't have access to client config which insulates them from potential false positives/negatives based on client-config-based behavior when it wasn't intended as such.